builds.go

```diff
@@ -98,8 +151,25 @@ func resolveRemoteArch(ctx context.Context, img string, arch string) (*RemoteRes
 	if cache == nil {
 		return nil, nil
 	}
-	r := cache.r
-	rArches := cache.arches
+
+	r := cache.Resolved
+	rArches := cache.Arches
+
+	if !wasCached {
+		fmt.Fprintf(os.Stderr, "NOTE: lookup %s -> %s\n", img, r.Desc.Digest)
```

Comment:
Is this a debugging statement or indented to remain?

Reply:
It's a bit of both -- I added it for debugging, but realized it was useful enough to know when lookups _actually_ happen that I left it in. :sweat_smile: